about.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-B899BJXW3P"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer `arrow functions ` over normal functions, so as to keep your _function_ syntax simpler and cleaner.

Reply:
It looks like google analytics requires you to using the standard function syntax overusing arrow function so I don't think I can change this without probably causing problems with google analytics.